packages/@uppy/companion-client/src/Provider.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const RequestClient = require('./RequestClient')
+const tokenStorage = require('./token-storage')
```

Comment:
Most of the other Uppy client file names use camel case `tokenStorage.js` (see eg. @uppy/utils)